projects/ADuCM3029_demo_cn0435/Scripts/dcs_cn0414_utilities.py

```diff
@@ -0,0 +1,634 @@
+        labels = [
+            Fore.YELLOW + 'Channel 1 MSW', Fore.YELLOW + 'Channel 1 LSW',
+            Fore.YELLOW + 'Channel 2 MSW', Fore.YELLOW + 'Channel 2 LSW',
+            Fore.YELLOW + 'Channel 3 MSW', Fore.YELLOW + 'Channel 3 LSW',
+            Fore.YELLOW + 'Channel 4 MSW', Fore.YELLOW + 'Channel 4 LSW',
+            Fore.YELLOW + 'Channel 5 MSW', Fore.YELLOW + 'Channel 5 LSW',
+            Fore.YELLOW + 'Channel 6 MSW', Fore.YELLOW + 'Channel 6 LSW',
+            Fore.YELLOW + 'Channel 7 MSW', Fore.YELLOW + 'Channel 7 LSW',
+            Fore.YELLOW + 'Channel 8 MSW', Fore.YELLOW + 'Channel 8 LSW',
+            Fore.YELLOW + 'Channel 1 OWD', Fore.YELLOW + 'Channel 2 OWD',
+            Fore.YELLOW + 'Channel 3 OWD', Fore.YELLOW + 'Channel 4 OWD',
+            Fore.YELLOW + 'HART IN 01', Fore.YELLOW + 'HART IN 02',
+            Fore.YELLOW + 'HART IN 03', Fore.YELLOW + 'HART IN 04',
+            Fore.YELLOW + 'HART IN 05', Fore.YELLOW + 'HART IN 06',
+            Fore.YELLOW + 'HART IN 07', Fore.YELLOW + 'HART IN 08',
+            Fore.YELLOW + 'HART IN 09', Fore.YELLOW + 'HART IN 10',
+            Fore.YELLOW + 'HART IN 11', Fore.YELLOW + 'HART IN 12',
+            Fore.YELLOW + 'HART IN 13', Fore.YELLOW + 'HART IN 14',
+            Fore.YELLOW + 'HART IN 15', Fore.YELLOW + 'HART IN 16',
+            Fore.YELLOW + 'HART IN 17', Fore.YELLOW + 'HART IN 18',
+            Fore.YELLOW + 'HART IN 19', Fore.YELLOW + 'HART IN 20',
+            Fore.YELLOW + 'HART IN 21', Fore.YELLOW + 'HART IN 22',
+            Fore.YELLOW + 'HART IN 23', Fore.YELLOW + 'HART IN 24',
+            Fore.YELLOW + 'HART IN 25', Fore.YELLOW + 'HART IN 26',
+            Fore.YELLOW + 'HART IN 27', Fore.YELLOW + 'HART IN 28',
+            Fore.YELLOW + 'HART IN 29', Fore.YELLOW + 'HART IN 30']
+        utilities.print_table(
+            register_address, registers_number, labels, registers)
+    return registers
+
+
+def read_board_current_channels(global_data: dict) -> None:
+    """Read a single board current channels.
+
+    Args:
+        global_data: Dictionary with global variables
+
+    Returns:
+        None
+
+    """
+    if '0' not in global_data["BOARDS"]:
+        print(Fore.LIGHTMAGENTA_EX + "No analog input board available at",
+              Fore.CYAN + "MODBUS address",
+              Fore.CYAN + str(global_data["MODBUS_ADDRESS"]))
+    else:
+        cs_address = request_adc_cs(global_data)
+        while True:
+            try:
+                currents = read_current_channels(global_data, cs_address)
+                print(Fore.GREEN + " ".join(
+                    str('{:+.5f} A'.format(element)) for element in currents))
+            except KeyboardInterrupt:
+                break
+            except OSError as error_message:
+                print(Fore.RED + 'Timeout error!', error_message)
+            except ValueError as error_message:
+                print(Fore.RED + 'Register operation error!', error_message)
+
```

Comment:
same as above

Reply:
Two lines are necessary to respect python linting.

Reply:
ack, not familiar with python code.